dbc2val/requirements.txt

```diff
@@ -1,8 +1,8 @@
 #
-# This file is autogenerated by pip-compile with Python 3.8
+# This file is autogenerated by pip-compile with Python 3.10
 # by the following command:
 #
-#    pip-compile requirements.in
+#    pip-compile --pre requirements.in
```

Comment:
this needs to be reverted back once we have an official release of kuksa-client. Do we remember this or better leave it as is?

Reply:
This file is automatically generated, but you are correct, as soon as we have a released kuksa-client we shall update *.in/*.txt files. I tried to write something at https://github.com/eclipse/kuksa.val.feeders/wiki/Release-Process